Import error using Python 3.11.0

I try using this library in Python 3.11.0 and I get an import error:
```
    import exrex
../../../env/lib/python3.11/site-packages/exrex.py:25: in <module>
    from re import sre_parse, U
E   ImportError: cannot import name 'sre_parse' from 're'
```

--- a/exrex.py
+++ b/exrex.py
@@ -22,7 +22,11 @@ try:
     from future_builtins import map, range
 except:
     pass
-from re import match, sre_parse, U
+from re import match, U
+try:
+    import re._parser as sre_parse
+except ImportError: # Python < 3.11
+    from re import sre_parse
 from itertools import tee
 from random import choice, randint
 from types import GeneratorType

--- a/tests.py
+++ b/tests.py
@@ -20,8 +20,12 @@
 
 from exrex import generate, count, getone, CATEGORIES, simplify
 import re
-import sre_parse
+try: 
+    import re._parser as sre_parse
+except ImportError: # Python < 3.11
+    from re import sre_parse
 from sys import exit, version_info
+
 IS_PY3 = version_info[0] == 3
 
 RS = {

